.github/workflows/deployment-images.yaml

```diff
@@ -5,6 +5,18 @@ on:
       build-information-path:
         required: true
         type: string
+      docker-build-dir:
```

Comment:
I think it would be clearer to name this parameter `docker-build-context`.

Reply:
docker-build-context is good